### PR TITLE
Lesson 1 4

### PR DESCRIPTION
**Pull request by gsyoung2020**
lesson 4 completed

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,35 +1,59 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import * as React from "react";
+import TodoList from "./TodoList.jsx";
+import AddTodoForm from "./AddTodoForm.jsx";
+import { useState } from "react";
+import Quickmaths from "./Quickmaths.jsx";
```
Comment (slamflipstrom) [inline src/App.jsx:5]: Hey Grant, it looks to me like you can remove this import altogether unless you intend to use it. I see that it's commented out below though. 

It's currently giving me errors when I try to run your code locally.

`[plugin:vite:import-analysis] Failed to resolve import "./Quickmaths.jsx" from "src/App.jsx". Does the file exist?`

image

File: src/AddTodoForm.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+/* eslint-disable no-undef */
+/* eslint-disable react/prop-types */
+import { useState } from "react";
+export default function AddTodoForm({ onAddTodo }) {
+  const [todoTitle, setTodoTitle] = useState("");
+
+  const handleTitleChange = (event) => {
+    var newTodoTitle = event.target.value;
+    setTodoTitle(newTodoTitle);
+    //event.target.elements.title.value
```
Comment (slamflipstrom) [inline src/AddTodoForm.jsx]: ✂️ you can delete this comment if you're not intending to use it

**Review by slamflipstrom — COMMENTED** [review]
Looks like you're on the right track here, Grant. That said, I think you still have a few things to clean up. Please let me know once you've gotten the error addressed and I will review again :).

File: src/App.jsx (hunk 1 above)
Comment (gsyoung2020) [reply src/App.jsx:5]: it was a file i created so i could test out doing props, originally i was trying to make a separate branch for testing out concepts outside of the required assignment, i just added it to the git-ignore, but if you could help me separate it into a different branch next week would be awesome. I added the file back though so it should no longer error

File: src/App.jsx [hunk 3]
```diff
@@ -1,35 +1,61 @@
...
+
+// const title = 'React';
 
 function App() {
-  const [count, setCount] = useState(0)
+  const [colorPicked, pickedColor] = useState("");
+  const [todoList, setTodoList] = useState([]);
+
+  const addTodo = (newTodo) => {
+    setTodoList([...todoList, newTodo]);
```
Comment (slamflipstrom) [inline src/App.jsx:14]: nice usage of spread here 👍

**Review by slamflipstrom — APPROVED** [review]
Nice work, Grant!

good_job

**Pull request merged**